src/utils/fhir.js

```diff
@@ -76,10 +80,13 @@ async function getResources(server, resourceType) {
  * @param {string} subscriptionTopic - R5 backport subscription topic
  * @returns a R5 Backport Subscription
  */
-function generateSubscription(criteria, url, token, subscriptionTopic = undefined) {
+function generateSubscription(criteria, url, token, subscriptionTopic = undefined, id = undefined) {
+  // Add regex to check if criteria has at least one search param since HAPI is expecting this
+  const criteriaReg = /(.+)(\?.)+/;
   const subscription = {
+    ...(id && { id }),
```

Comment:
Thoughts on making this:
```suggestion
    id: id ?? shortId.generate(),
```

Reply:
Or since we already have uuid as a package we could 
```
const { v4: uuidv4 } = require('uuid');
...
id: id ?? uuidv4(),
```

Reply:
Made this change ``id: id ?? `sub${uuidv4()}`,`` in case we wanted to pass the id to HAPI.